gems/pending/fs/ext4/superblock.rb

```diff
@@ -223,6 +223,14 @@ def inodeSize
       isDynamic? ? @sb['inode_size'] : INODE_SIZE
     end
 
+    def isEnabled64Bit?
+      gotBit?(@sb['incompat_flags'], ICF_64BIT)
```

Comment:
If there's a chance we'll call this multiple times, we my want to memoize the value:
```
@is_enabled_64_bit ||= gotBit?(@sb['incompat_flags'], ICF_64BIT)
```